09:48:32 [ERR][      Workbench]: Console redirection failed.
09:48:32 [INF][      Workbench]: Starting up Workbench
09:48:32 [INF][      Workbench]: Current environment:
	Command line: "C:\Program Files\MySQL\MySQL Workbench 8.0 CE\MySQLWorkbench.exe" "C:\Users\luizl\AppData\Local\Temp\Rar$DIa12660.39213\Script Banco BDVendas.sql"
	CurrentDirectory: C:\WINDOWS\system32
	HasShutdownStarted: False
	OSVersion: Microsoft Windows NT 6.2.9200.0
	SystemDirectory: C:\WINDOWS\system32
	TickCount: 646156578
	UserInteractive: True
	Version: 4.0.30319.42000
	WorkingSet: 38281216
09:48:32 [INF][      Workbench]: Environment variables:
	SystemDrive = C:
	ProgramFiles(x86) = C:\Program Files (x86)
	ProgramW6432 = C:\Program Files
	PROCESSOR_IDENTIFIER = AMD64 Family 23 Model 1 Stepping 1, AuthenticAMD
	VBOX_MSI_INSTALL_PATH = C:\Program Files\Oracle\VirtualBox\
	TMP = C:\Users\luizl\AppData\Local\Temp
	PROCESSOR_ARCHITECTURE = AMD64
	Path = C:\Program Files\WinRAR;C:\Program Files\Google\Chrome\Application;C:\Program Files\Common Files\Oracle\Java\javapath;C:\Program Files (x86)\Common Files\Oracle\Java\javapath;C:\WINDOWS\system32;C:\WINDOWS;C:\WINDOWS\System32\Wbem;C:\WINDOWS\System32\WindowsPowerShell\v1.0\;C:\WINDOWS\System32\OpenSSH\;C:\Users\luizl\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.11_qbz5n2kfra8p0;C:\Users\luizl\AppData\Local\Microsoft\WindowsApps;;C:\Users\luizl\AppData\Local\Programs\Microsoft VS Code\bin
	HOMEPATH = \Users\luizl
	PROCESSOR_REVISION = 0101
	TEMP = C:\Users\luizl\AppData\Local\Temp
	USERPROFILE = C:\Users\luizl
	CommonProgramFiles(x86) = C:\Program Files (x86)\Common Files
	LOGONSERVER = \\DESKTOP-RG0LA1C
	USERNAME = luizl
	SystemRoot = C:\WINDOWS
	CHROME_CRASHPAD_PIPE_NAME = \\.\pipe\crashpad_14472_PTNOZPERIXGXTBKE
	OneDrive = C:\Users\luizl\OneDrive
	CommonProgramFiles = C:\Program Files\Common Files
	PROCESSOR_LEVEL = 23
	ProgramData = C:\ProgramData
	RlsSvcPort = 22112
	CHROME_RESTART = Google Chrome|Nossa! Houve uma falha no Google Chrome. Reiniciar agora?|LEFT_TO_RIGHT
	OneDriveConsumer = C:\Users\luizl\OneDrive
	COMPUTERNAME = DESKTOP-RG0LA1C
	ALLUSERSPROFILE = C:\ProgramData
	CommonProgramW6432 = C:\Program Files\Common Files
	SESSIONNAME = Console
	DriverData = C:\Windows\System32\Drivers\DriverData
	HOMEDRIVE = C:
	windir = C:\WINDOWS
	ACSetupSvcPort = 23210
	NUMBER_OF_PROCESSORS = 16
	OS = Windows_NT
	ProgramFiles = C:\Program Files
	ComSpec = C:\WINDOWS\system32\cmd.exe
	PATHEXT = .COM;.EXE;.BAT;.CMD;.VBS;.VBE;.JS;.JSE;.WSF;.WSH;.MSC
	JAVA_HOME = JDK
	PSModulePath = C:\Program Files\WindowsPowerShell\Modules;C:\WINDOWS\system32\WindowsPowerShell\v1.0\Modules
	APPDATA = C:\Users\luizl\AppData\Roaming
	USERDOMAIN = DESKTOP-RG0LA1C
	ACSvcPort = 17532
	LOCALAPPDATA = C:\Users\luizl\AppData\Local
	USERDOMAIN_ROAMINGPROFILE = DESKTOP-RG0LA1C
	PUBLIC = C:\Users\Public
09:48:32 [INF][      Workbench]: Current version given by meta info is: 8.0.34
09:48:32 [INF][      Workbench]: Exiting as another instance of WB is already running.
09:48:32 [INF][      WBContext]: Opening SQL Editor window to ''...
09:48:33 [INF][  column_widths]: Initializing column width cache for unconnected
09:48:33 [INF][        WQE.net]: Launching SQL IDE
09:48:33 [INF][        WQE.net]: SQL IDE UI is ready
09:49:03 [INF][        WQE.net]: Shutting down SQL editor (unconnected)
09:49:06 [ERR][wb_server_management.py:local_run_cmd_windows:407]: Exception executing local command: chcp.com: 'utf-8' codec can't decode byte 0xa0 in position 1: invalid start byte
Traceback (most recent call last):
  File "C:\Program Files\MySQL\MySQL Workbench 8.0 CE\modules\wb_server_management.py", line 404, in local_run_cmd_windows
    retcode = OSUtils.exec_command(command, output_handler)
              ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Program Files\MySQL\MySQL Workbench 8.0 CE\workbench\os_utils.py", line 360, in exec_command
    for line in iter(process.stdout.readline, ""):
  File "<frozen codecs>", line 322, in decode
UnicodeDecodeError: 'utf-8' codec can't decode byte 0xa0 in position 1: invalid start byte

09:49:06 [WRN][wb_server_management.py:fetch_windows_shell_info:654]: ProcessOpsWindowsLocal.fetch_windows_shell_info(): WARNING: Unable to determine codepage from shell: ""
09:49:06 [ERR][wb_server_management.py:local_run_cmd_windows:407]: Exception executing local command: ver: 'utf-8' codec can't decode byte 0xc6 in position 23: invalid continuation byte
Traceback (most recent call last):
  File "C:\Program Files\MySQL\MySQL Workbench 8.0 CE\modules\wb_server_management.py", line 404, in local_run_cmd_windows
    retcode = OSUtils.exec_command(command, output_handler)
              ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Program Files\MySQL\MySQL Workbench 8.0 CE\workbench\os_utils.py", line 360, in exec_command
    for line in iter(process.stdout.readline, ""):
  File "<frozen codecs>", line 322, in decode
UnicodeDecodeError: 'utf-8' codec can't decode byte 0xc6 in position 23: invalid continuation byte

09:49:34 [ERR][       pymforms]: Unhandled exception in Python code: 
Traceback (most recent call last):
  File "C:\Program Files\MySQL\MySQL Workbench 8.0 CE\modules\wb_admin_grt.py", line 303, in _sidebar_entry_clicked
    self.open_into_section(entry_id)
  File "C:\Program Files\MySQL\MySQL Workbench 8.0 CE\modules\wb_admin_grt.py", line 329, in open_into_section
    page = page_class(self.ctrl_be, self.server_profile, self.admin_tab)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Program Files\MySQL\MySQL Workbench 8.0 CE\modules\wb_admin_server_status.py", line 234, in __init__
    self.status = wb_admin_monitor.WbAdminMonitor(server_profile, self.ctrl_be)
                  ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Program Files\MySQL\MySQL Workbench 8.0 CE\modules\wb_admin_monitor.py", line 251, in __init__
    self.mon_be = wba_monitor_be.WBAdminMonitorBE(UPDATE_INTERVAL, server_profile, ctrl_be, self.widgets, self.cpu_widget, sql)
                  ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Program Files\MySQL\MySQL Workbench 8.0 CE\modules\wba_monitor_be.py", line 398, in __init__
    self.wmimon = WMIStats(ctrl_be, server_profile, cpu_widget)
                  ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Program Files\MySQL\MySQL Workbench 8.0 CE\modules\wba_monitor_be.py", line 343, in __init__
    raise Exception("Current profile has no WMI enabled") # TODO Should be better message
    ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
Exception: Current profile has no WMI enabled
09:53:50 [ERR][ sqlide-history]: Can't open SQL history file C:\Users\luizl\AppData\Roaming\MySQL\Workbench\sql_history\2023-10-12
09:53:50 [ERR][SQL Editor Form]: Error Code: 1133. Can't find any matching row in the user table
10:23:03 [INF][        WQE.net]: Shutting down SQL editor (vendas)
